### PR TITLE
give better names to graphql queries

### PR DESCRIPTION
**Pull request by Xyzrr**
Description
-----------
Gives more unique and descriptive names to GraphQL queries for better monitoring.

My Retool dashboard shows that Model is both the most popular and most problematic GraphQL query Gorilla receives. But after investigating, I realized there's 6 different queries from the CLI all named "Model", so this aggregate statistic is useless.

Screen Shot 2022-01-05 at 2 58 37 PM

Testing
-------
How was this PR tested?

Checklist
-------
- Name PR "[WB-NNNN][WB-MMMM] Add support for..." similar to entries in CHANGELOG.md
- Include reference to internal ticket "Fixes WB-NNNN" (and github issue "Fixes #NNNN" if applicable)

### CODE REVIEW

File: functional_tests/imports/01-batch1.py [hunk 1]
```diff
@@ -8,7 +8,9 @@
 plugin:
   - wandb
 depend:
-  pip_install_timeout: 1500  # 25m
+  pip_install_timeout: 3000  # 50m
```
Comment (kptkin) [inline functional_tests/imports/01-batch1.py:11]: this PR should fix your issue with imports test:
https://github.com/wandb/client/pull/3167

File: tox.ini [hunk 2]
```diff
@@ -351,7 +351,7 @@ deps =
     -r{toxinidir}/requirements.txt
     func-s_base-py{35,36,37,38,39,310}: -r{toxinidir}/requirements_dev.txt
     pytest-mock
-    yea-wandb==0.7.28
+    git+https://github.com/wandb/yea-wandb.git@jq1/rename-queries#egg=yea-wandb
```
Comment (kptkin) [inline tox.ini:354]: before merging could you update `yea-wandb` with your new changes and pin to the relevant version? 
I could help publish it

File: functional_tests/imports/01-batch1.py (hunk 1 above)
Comment (Xyzrr) [reply functional_tests/imports/01-batch1.py:11]: thank you!

**Review by raubitsj — APPROVED** [review]
LGTM,  can we get a quick audit from someone else in the backend, specifically to ok the changes due to bypassing relay.go logic and modifying the required parameters. 

https://github.com/wandb/core/blob/053cb8e24c6d72db6882b5fab613749f8ef1ece8/services/gorilla/handler/relay.go

**Comment by dannygoldstein** [comment]
LGTM, but older client versions wont' have these changes propagated and will still report the graphql queries that are changed in this PR under their old names. Will this be a problem for monitoring or are we okay to have a period where the same graphql requests are logged under different names by gorilla depending on the client verison?

**Comment by Xyzrr** [comment]
> LGTM, but older client versions wont' have these changes propagated and will still report the graphql queries that are changed in this PR under their old names. Will this be a problem for monitoring or are we okay to have a period where the same graphql requests are logged under different names by gorilla depending on the client verison?

That's a good point, there'll be a period of mild confusion for people who don't know the history. But this is definitely the way we want to go long-term so I think it's worth.

**Pull request merged**